foursight_core/run_result.py

```diff
@@ -161,14 +173,19 @@ def record_run_info(self):
         Add a record of the completed check to the runs bucket with name
         equal to the dependency id. The object itself is only the status of the run.
         Returns True on success, False otherwise
+
+        NOTE: This functionality is sort of useless I'd say - we don't create a special bucket
+        for it anyway, so I am disabling it and replacing the run info with log statement that will
+        propagate in the check runner - so you can always search for record_run_info to find these. - Will Nov 15 2022
```

Comment:
Slightly weird that this comment is in doc string, but I guess it's OK in this case.

Should the function just go away? Do you plan to  ultimately do more? Is there a compatibility issue removing it?

Reply:
I don't want to remove it entirely as it can provide a record in case a check doesn't actually make it into the datastore (ie: serialization error, seen commonly with datetime, fixed now though)